### PR TITLE
Upgrade to 3.8.4

### PR DESCRIPTION
**Pull request by saraedum**
Fixes #19
Fixes #21

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@isuruf any idea where this one is coming from:
```
lld-link: warning: ignoring unknown argument '-rpath'
lld-link: error: could not open 'D:/bld/normaliz_1590582882765/_h_env/Library/lib': is a directory
```

**Comment by isuruf** [comment]
[Here](https://github.com/Normaliz/Normaliz/blob/648f3767ecf2c5d50d496a36ff109a6fcc3c5da0/configure.ac#L221) and [here](https://github.com/Normaliz/Normaliz/blob/648f3767ecf2c5d50d496a36ff109a6fcc3c5da0/configure.ac#L265)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ requirements:
     - libtool              # [not win]
     - make                 # [not win]
   host:
-    - boost-cpp
+    - libflint             # [not win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: `libflint` is already listed below. Does this really not use Boost any more?

Comment (saraedum) [reply recipe/meta.yaml]: Yes, they told me they don't.

**Comment by saraedum** [comment]
Output precision has changed in recent versions of e-antic:
```
diff -b test-nf/2cone.ref test-nf/2cone.out > test-nf/2cone.diff
11c11
< resulting sum of |det|s = (a+2 ~ 3.414214)
---
> resulting sum of |det|s = (a+2 ~ 3.4142136)
19c19
< volume (lattice normalized) = (a+1 ~ 2.414214)
---
> volume (lattice normalized) = (a+1 ~ 2.4142136)
26c26
<  1 (a ~ 1.414214)
---
>  1 (a ~ 1.4142136)
30c30
<  (a ~ 1.414214) -1
---
>  (a ~ 1.4142136) -1
```

**Pull request merged**